PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -193,6 +223,21 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   o2::framework::OutputObj<ZorroSummary> zorroSummary{"zorroSummary"};
   int currentRun{-1};
 
+  /// Set standard preselection gap trigger (values taken from UD group)
+  void setSGPreselection(SGCutParHolder& sgCuts)
```

Comment:
Why do you pass `sgCuts`?

Reply:
Actually, this might be a better place to call `sgCuts = SGCutParHolder()` since it might not always be needed.